pkg/commands/config/canonical_service_account.go

```diff
@@ -0,0 +1,75 @@
+package config
+
+import (
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	corev1 "k8s.io/api/core/v1"
+
+	"github.com/vmware-tanzu/kpack-cli/pkg/commands"
+	"github.com/vmware-tanzu/kpack-cli/pkg/config"
+	"github.com/vmware-tanzu/kpack-cli/pkg/k8s"
+)
+
+func NewCanonicalServiceAccountCommand(clientSetProvider k8s.ClientSetProvider) *cobra.Command {
+	var namespace string
+
+	cmd := &cobra.Command{
+		Use:          "canonical-service-account [name]",
+		Short: "Set or Get the canonical service account",
+		Long: `Set or Get the canonical service account 
+
+The canonical service account is the service account that contains the secrets for communicating with the canonical repository. It is required to be configured to use kp.
```

Comment:
Something like?
`The canonical service account will be set as the service account on all cluster builders created with kp and the secrets on the service account will used to provide credentials to write cluster builder images.`

Reply:
would you replace line 21 with that sentence or just add that sentence